src/main/java/com/iridium/iridiumskyblock/database/Island.java

```diff
@@ -71,6 +74,16 @@ public Island(@NotNull String name, @NotNull Schematics.SchematicConfig schemati
         this.home = schematicConfig.xHome + "," + schematicConfig.yHome + "," + schematicConfig.zHome + ",0,0";
         this.time = ZonedDateTime.of(LocalDateTime.now(), ZoneId.systemDefault()).toInstant().toEpochMilli();
         this.color = Color.BLUE;
+        this.experience = 0;
+    }
+
+    /**
+     * Gets the island's level
+     *
+     * @return The islands level
+     */
+    public int getLevel() {
+        return (int) Math.cbrt(experience + 1);
```

Comment:
This is a pretty bad way to calculate the level IMO

Reply:
There are multiple reasons for this.
First off, Math.cbrt() is very slow by a large margin even in comparison to other functions like Math.sqrt() which are already considered to be too slow: https://replit.com/@Dasunterstrich/InternationalSeagreenRom
Next, the graph for this function is simply not good for levels. The first three levels only need 99 XP, while level 10 already requires 1000, which in my opinion is too steep / a too big difference.

Reply:
ye I didnt really know what to do for levels feel free to replace it with another function at a different time, but for now ima just leave it for testing purposes

Reply:
Also its not that big of a deal according to your code cube rooting 1000000 times only takes 130ms, shouldnt be that big of a deal